yarn-project/txe/src/txe_service/txe_service.ts

```diff
@@ -874,7 +874,7 @@ export class TXEService {
     return toForeignCallResult(secret.toFields().map(toSingle));
   }
 
-  emitOffchainMessage(_message: ForeignCallArray, _recipient: ForeignCallSingle) {
+  emitOffchainEffect(_data: ForeignCallArray) {
     if (!this.oraclesEnabled) {
       throw new Error(
         'Oracle access from the root of a TXe test are not enabled. Please use env._ to interact with the oracles.',
```

Comment:
The comment on line 884 needs updating. Would search for all the occurrences of "offchain message" in the codebase to see if there are some stale comments remaining.